Kinvey.Core/Offline/SQLiteCacheManager.cs

```diff
@@ -49,6 +59,7 @@ private SQLiteConnection DBConnectionSync
 					//var connectionFactory = new Func<SQLiteConnectionWithLock>(()=>new SQLiteConnectionWithLock(platform, new SQLiteConnectionString(this.dbpath, false, null, new KinveyContractResolver())));
 					//dbConnection = new SQLiteAsyncConnection (connectionFactory);
 					_dbConnectionSync = new SQLiteConnection(platform, dbpath, false, null, null, null, new KinveyContractResolver());
+                    //_dbConnectionSync.TraceListener = new DebugTraceListener();
```

Comment:
@heyzooi  is the idea to uncomment this for tracing? How about adding (in addition to this) a debug log for SQL statements? We do have logging levels in the SDK. With a debug log, a customer could potentially see all SQL interactions as well.